src/components/Firestore/Collection.tsx

```diff
@@ -68,14 +69,14 @@ export const Collection: React.FC<Props> = ({ collection }) => {
       <Route
         path={`${url}/:id`}
         render={({ match }: any) => {
-          const doc = docs.find(d => d.ref.id === match.params.id);
+          const docRef = collection.doc(match.params.id);
 
-          if (!doc) {
+          if (!docRef) {
```

Comment:
Remove this if-statement -- it cannot be null.

Reply:
Done